.travis/dispatch.sh

```diff
@@ -72,4 +72,5 @@ else
     if [[ "${SALT_NODE_ID}" == "servo-master1" ]]; then
         ./test.py sls.buildbot.master sls.homu sls.nginx
     fi
+    ./test.py sls.common
```

Comment:
We should only run this on the non-macOS machines for now, so we can check the `${SALT_NODE_ID}` isn't a macOS node; easiest way is probably using the `=~` bash operator.